Neptune.WebMvc/Views/DataHub/IndexViewData.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Neptune.EFModels.Entities;
+using Neptune.WebMvc.Common;
+using Neptune.WebMvc.Controllers;
+using Neptune.WebMvc.Models;
+using Neptune.WebMvc.Security;
+using Neptune.WebMvc.Views.Shared;
+
+namespace Neptune.WebMvc.Views.DataHub
+{
+    public class IndexViewData : NeptuneViewData
+    {
+        public bool HasManagePermission { get; }
+        public readonly WebServiceToken WebServiceAccessToken;
+        public readonly List<WebServiceDocumentation> ServiceDocumentationList;
+        public ViewPageContentViewData TreatmentBMPPage { get; }
+        public ViewPageContentViewData DelineationPage { get; }
+        public ViewPageContentViewData FieldTripPage { get; }
+        public ViewPageContentViewData WQMPPage { get; }
+        public ViewPageContentViewData SimplifiedBMPPage { get; }
+        public ViewPageContentViewData WQMPLocationPage { get; }
+        public ViewPageContentViewData AssessmentAreaPage { get; }
+        public ViewPageContentViewData OVTAsPage { get; }
+        public ViewPageContentViewData LandUseBlocksPage { get; }
+        public ViewPageContentViewData ParcelPage { get; }
+        public ViewPageContentViewData RegionalSubbasinsPage { get; }
+        public ViewPageContentViewData LandUseStatisticsPage { get; }
+        public ViewPageContentViewData ModelBasinsPage { get; }
+        public ViewPageContentViewData PrecipitationZonesPage { get; }
+        public string UploadTreatmentBMPUrl { get; set; }

```

Comment:
please remove the setters in all these; these properties should be immutable and only set via the constructor, so only getters